AppCenterData/AppCenterData/Internal/LocalStore/MSDBDocumentStore.m

```diff
@@ -263,4 +363,73 @@ + (NSString *)tableNameForPartition:(NSString *)partition {
   return pendingDocuments;
 }
 
+- (BOOL)hasPendingOperationsForPartition:(NSString *)partition {
+  NSString *tableName = [MSDBDocumentStore tableNameForPartition:partition];
+  NSString *selectionQuery =
+      [NSString stringWithFormat:@"SELECT count(*) FROM \"%@\" WHERE \"%@\" IS NOT NULL", tableName, kMSPendingOperationColumnName];
+
+  NSArray<NSArray<NSNumber *> *> *result = [self.dbStorage executeSelectionQuery:selectionQuery];
```

Comment:
Why do you use generic `NSArray<NSNumber *>` array as a return type here but `NSArray *` earlier, while the method itself returns `NSArray<NSArray *> *`? I think we need some consistency here.

Reply:
That `NSArray *` earlier if it's referring to `NSArray *listResult` is for a single row that contains multiple types so it makes sense to be generic. The method returns a generic parameter because it can't make assumption on how it'll be used, what query...But in this code since the the query is a count we now we are expecting a number so that sounds correct to me. :)